scripts/vcpkg_ports/opencascade/portfile.cmake

```diff
@@ -0,0 +1,89 @@
+string(REPLACE "." "_" VERSION_STR "V${VERSION}")
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Open-Cascade-SAS/OCCT
+    REF "${VERSION_STR}"
+    SHA512 af176cbd105c49949282c16bb6e30f69167bc6c00a50e0ae69aea555815d47ac3c4540c233e596c5add7cb846c2b33d7be267d8e02472286e758b662b4a652ab
+    HEAD_REF master
+    PATCHES
+        dependencies.patch
+        drop-bin-letter-d.patch
+        fix-pdb-find.patch
+        fix-install-prefix-path.patch
+        install-include-dir.patch
+)
+
+if (VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
+    set(BUILD_TYPE "Shared")
+else()
+    set(BUILD_TYPE "Static")
+endif()
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        freeimage   USE_FREEIMAGE
+        freetype    USE_FREETYPE
+        rapidjson   USE_RAPIDJSON
+        samples     INSTALL_SAMPLES
+        tbb         USE_TBB
+        vtk         USE_VTK
+)
+
+### BEGIN AXOM PATCH
+
+# replace a quoted number in a flag -- /wd"26812" with /wd26812
+set(_file "${SOURCE_PATH}/adm/cmake/occt_defs_flags.cmake")
+file(READ "${_file}" _filedata)
+string(REPLACE [[/wd\"26812\"]] [[/wd26812]] _filedata "${_filedata}")
+file(WRITE "${_file}" "${_filedata}")
+### END AXOM PATCH
```

Comment:
Note: I had to copy the whole `opencascade` portfile to add this patch, which converts `/wd"26812"` to `/wd26812`